### DISCUSSION TITLE
What is the name of the font that qutebrowser uses?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#6536] [Q&A] [answered]
I think this font looks really nice but in qutebrowser all it says is (default font). I'd like to know because it looks really nice.

**Comment by toofar** [comment, marked as answer]
Probably your default font! As in I don't think it ships with any fonts so it should come from your system.
What part of qutebrowser is this particular font used in? Where are you seeing this (default font)?

**Reply by The-Compiler** [reply]
Indeed it's just using whatever Qt gives us as a system-wide fixed font.

You should get the same manually by running `python3 -c "from PyQt5.QtWidgets import QApplication; from PyQt5.QtGui import QFontDatabase; a = QApplication([]); print(QFontDatabase.systemFont(QFontDatabase.FixedFont).family())"`

**Reply by ghost** [reply]
@toofar In the names of tabs and on the status bar. 

@The-Compiler it's monospace, but the thing is I have my terminal font as monospace too, but it doesn't look the same? Maybe I'm just weird.

**Reply by The-Compiler** [reply]
No idea then... Might be something Qt's font selection/rendering does differently from your terminal.
